<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSRMTableDialog</class>
 <widget class="QDialog" name="OSRMTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSRM: Get a time-distance matrix with OSRM</string>
  </property>
  <widget class="QPushButton" name="pushButton_fetch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fetch and save the result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>410</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>About..</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="close_button_box">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>93</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Source point layer:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QgsMapLayerComboBox" name="comboBox_layer"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Source ID field:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QgsFieldComboBox" name="comboBox_idfield"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Destination point layer:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QgsMapLayerComboBox" name="comboBox_layer_2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Destination ID field:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QgsFieldComboBox" name="comboBox_idfield_2"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_minutes">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Time converted in minutes</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_flatten">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Flatten the matrix </string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_empty_val">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Put an empty value for not found connection</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select *.csv output file:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="pushButton_browse">
      <property name="text">
       <string>Browse</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_output">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Provider:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_provider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>5</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select the source point layer and id field:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>destination point layer is optional</string>
   </property>
  </widget>
  <zorder>pushButton_about</zorder>
  <zorder>close_button_box</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_fetch</zorder>
  <zorder>label_8</zorder>
  <zorder>comboBox_provider</zorder>
  <zorder>label_9</zorder>
  <zorder>label_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_button_box</sender>
   <signal>accepted()</signal>
   <receiver>OSRMTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_button_box</sender>
   <signal>rejected()</signal>
   <receiver>OSRMTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
